internal/controllers/utils/utils.go

```diff
@@ -198,6 +203,21 @@ func GetDeploymentVolumes(serverName string) []corev1.Volume {
 		}
 	}
 
+	if HasDatabase(serverName) || serverName == InventoryDatabaseServerName {
```

Comment:
why do we need this volume for `InventoryDatabaseServerName`?

Reply:
It is required by the init script that creates the users and logical databases.

Reply:
I see, so this was not working before, right? This is not part of the migration init container

Reply:
Let me take another look at this.

Reply:
Sorry, not sure what I was thinking there... I've removed it as it was unnecessary.  Thanks for catching this.